src/swap/changehero.ts

```diff
@@ -43,6 +43,7 @@ const asInitOptions = asObject({
   apiKey: asString
 })
 
+// See https://changenow.io/currencies for list of supported currencies
```

Comment:
This file is for ChangeHero, but the link is for ChangeNow?

Reply:
Oh, I see the very next commit fixes it. The next commit should be a fixup / squash  then.

Reply:
I squashed all the URI changes to one commit.